### PR TITLE
first start

### PR DESCRIPTION
**Pull request by olhry**
- [DEMO LINK]
https://olhry.github.io/layout_catalog/
- [TEST REPORT LINK]
https://olhry.github.io/layout_catalog/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,12 +1,451 @@
...
+        <ul class="header__list">
+          <li class="header__item">
+            <a
+              href="#apple"
+              class="header__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__item">
```
Comment (Anastasiia-Svintsova) [inline src/index.html:36]: Don't forget to add empty lines between multiline sibling blocks of HTML
```suggestion
          </li>
          
          <li class="header__item">
```

File: src/index.html [hunk 2]
```diff
@@ -1,12 +1,451 @@
...
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div class="card" data-qa="card">
```
Comment (Anastasiia-Svintsova) [inline src/index.html:98]: Follow the BEM, please 
```suggestion
      <div class="catalog__card card" data-qa="card">
```

File: src/styles/blocks/card.scss [hunk 3]
```diff
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
```
Comment (Anastasiia-Svintsova) [inline src/styles/blocks/card.scss:2]: Are you sure you need a display: flex property for the card?

File: src/styles/blocks/card.scss [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  background-color: $color-white;
+  border: 1px solid $color-light;
+  padding: 32px 16px 16px ;
+
+  &__image {
+    height: 134px;
```
Comment (Anastasiia-Svintsova) [inline src/styles/blocks/card.scss:13]: Add display: block; to the image to apply sizes correctly

File: src/styles/blocks/card.scss [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    text-decoration: none;
+    justify-content: center;
+    align-items: center;
+    color: $color-white;
+    background-color: $color-blue;
+    border: 1px solid $color-blue;
+    border-radius: 5px;
+  }
+
+  transition: all 300ms;
```
Comment (Anastasiia-Svintsova) [inline src/styles/blocks/card.scss:86]: Don't use all for transition property. Specify the properties instead

File: src/styles/blocks/catalog.scss [hunk 6]
```diff
@@ -0,0 +1,33 @@
+.catalog {
+  --column-count: 1;
+  padding: 50px 40px 72px;
+  display: grid;
+  row-gap: 48px;
+  column-gap: 46px;
```
Comment (Anastasiia-Svintsova) [inline src/styles/blocks/catalog.scss:6]: - The gap should be 48px 
image

File: src/styles/blocks/catalog.scss [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    & {
+      padding: 50px 128px 72px;
+    }
+  }
+
+  @media (min-width: 488px) {
+    & {
+      --column-count: 2;
+    }
+  }
```
Comment (Anastasiia-Svintsova) [inline src/styles/blocks/catalog.scss:20]: You don't need to use media here 
image

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
- Make sure that all your elements have a consistent box model (box-sizing: border-box);
- Check the sizes for all elements. For example the `buy` button 
image
image

File: src/index.html (hunk 2 above)
Comment (olhry) [reply src/index.html:98]: Ok

File: src/styles/blocks/card.scss (hunk 3 above)
Comment (olhry) [reply src/styles/blocks/card.scss:2]: Так, цей блок з попереднього завдання з флекс

File: src/styles/blocks/card.scss (hunk 5 above)
Comment (olhry) [reply src/styles/blocks/card.scss:86]: Ok

File: src/styles/blocks/catalog.scss (hunk 6 above)
Comment (olhry) [reply src/styles/blocks/catalog.scss:6]: Виправив розміри і всі тести пройшли

File: src/styles/blocks/catalog.scss (hunk 7 above)
Comment (olhry) [reply src/styles/blocks/catalog.scss:20]: Не зовсім розумію. У завданні написано використовувати розміри з файлу grid.md А в ньому як раз всі розміри для медіа стоять 

# Catalog using grid layout
Implement the layout using `grid`
- Gap between cards should be: horizontal - `48px`, vertical - 46px
- Blocks should start from the left
- Media queries:
  - There should be four columns starting at `1024px`
  - There should be three columns starting at `768px`
  - There should be two columns starting at `488px`
  - There should be one column below `488px`

Comment (Anastasiia-Svintsova) [reply src/styles/blocks/catalog.scss:20]: sorry, my mistake

File: src/styles/blocks/card.scss [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+    text-decoration: none;
+    justify-content: center;
+    align-items: center;
+    color: $color-white;
+    background-color: $color-blue;
+    border: 1px solid $color-blue;
+    border-radius: 5px;
+  }
+
+  transition-duration: 300ms;
```
Comment (Anastasiia-Svintsova) [inline src/styles/blocks/card.scss:86]: What about the transition property?

File: src/styles/utils/variables.scss [hunk 9]
```diff
@@ -0,0 +1,7 @@
+$color-main: #060b35;
+$color-secondary: #616070;
+$color-light: #f3f3f3;
+$color-white: #fff;
+$color-blue: #00acdc;
+$logo-width: 40px;
+$logo-height: 40px;
```
Comment (Anastasiia-Svintsova) [inline src/styles/utils/variables.scss:7]:
```suggestion
$logo-size: 40px;
```

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
- Can't see these changes on hover 
image

File: src/styles/utils/variables.scss (hunk 9 above)
Comment (olhry) [reply src/styles/utils/variables.scss]: Змінив

**Review by lerastarynets — APPROVED** [review]
You're doing a fantastic job, keep going!

**Pull request closed without merging**